README.md

```diff
@@ -90,7 +90,14 @@ a GHC [bug][ghc-split-objs-bug], you need a full clean rebuild when using this f
 * `--verbose`: run Hadrian in verbose mode. In particular this prints diagnostic messages
 by Shake oracles.
 
-#### User settings
+#### Cross Compiling
+
+1. Set up and test as described in https://gist.github.com/izgzhen/26e49e7bf2e79a9a8f0a3ec37e4837af
```

Comment:
This doc should probably go into `doc/cross-compile.md`

Reply:
Ah, good idea